src/date-picker/src/panel/date.tsx

```diff
@@ -102,30 +103,35 @@ export default defineComponent({
           </div>
         </div>
         {this.actions?.length ? (
-          <div class={`${mergedClsPrefix}-date-panel-actions`}>
-            <div class={`${mergedClsPrefix}-date-panel-actions__prefix`}></div>
-            <div class={`${mergedClsPrefix}-date-panel-actions__suffix`}>
-              {this.actions.includes('clear') ? (
-                <NButton
-                  theme={mergedTheme.peers.Button}
-                  themeOverrides={mergedTheme.peerOverrides.Button}
-                  size="tiny"
-                  onClick={this.handleClearClick}
-                >
-                  {{ default: () => this.locale.clear }}
-                </NButton>
-              ) : null}
-              {this.actions.includes('now') ? (
-                <NButton
-                  theme={mergedTheme.peers.Button}
-                  themeOverrides={mergedTheme.peerOverrides.Button}
-                  size="tiny"
-                  onClick={this.handleNowClick}
-                >
-                  {{ default: () => this.locale.now }}
-                </NButton>
-              ) : null}
-              {/** we don't need a confirm button for date picking */}
+          <div class={`${mergedClsPrefix}-date-panel-footer`}>
```

Comment:
Footer and actions are independant rows. Do not wrap them in a div. Use css grid. It's already defined in cssr files.